project/profiles/capacity4more/modules/c4m/content/c4m_content_news/c4m_content_news.features.field_base.inc

```diff
@@ -0,0 +1,35 @@
+<?php
+/**
+ * @file
+ * c4m_content_news.features.field_base.inc
+ */
+
+/**
+ * Implements hook_field_default_field_bases().
+ */
+function c4m_content_news_field_default_field_bases() {
+  $field_bases = array();
+
+  // Exported field_base: 'body'.
+  $field_bases['body'] = array(
```

Comment:
Can you please use c4m_body instance, which is used on other content types, and remove the base definition?